### DISCUSSION TITLE
Scheda plancia fai da te

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#2136] [HUB: Home Assistant]
Mi scuso per il titolo ma non sapevo proprio cosa mettere.

Ho 3 termostati tado che controllano una stufa alimentata per mezzo di una presa smart.
Vorrei creare una scheda nella plancia che mostri quanto segue:

- per ogni termostato se questo si trovi o meno sullo stato "riscaldamento" ("heating" nello stato del controllo HAVC);
- per la presa smart se questa stia misurando o meno un potenza maggiore a 5W;

In sostanza vorrei sapere se la stufa è accesa o meno e al tempo stesso quale termostato è attivo. Vorrei però avere per ognuna della 4 voci un solo stato booleano ovvero SI e NO, non voglio vedere i W della presa o tutte le informazioni del controllo HAVC.

C'è un modo? Se si quale scheda va usata? Devo installare qualche componente custom?

**Comment by 50494554524F** [comment]
creati dei binary_sensor template...
